CHANGELOG-0.7.0.md

```diff
@@ -0,0 +1,13 @@
+# Changelog since v0.6.1
+
+## New Features
+
+- Introduce a CSI Metrics Library that can be used to automatically record prometheus metrics for every CSI gRPC call. ([#35](https://github.com/kubernetes-csi/csi-lib-utils/pull/35), [@saad-ali](https://github.com/saad-ali))
+
+
+## Other Notable Changes
```

Comment:
Also add a note that k8s dependencies have been updated to 1.17.0

Reply:
Done.